examples/sv1-client-and-server/Cargo.toml

```diff
@@ -12,3 +12,4 @@ async-std="1.8.0"
 serde = { version = "1.0.89", default-features = false, features = ["derive", "alloc"] }
 serde_json = { version = "1.0.64", default-features = false, features = ["alloc"] }
 async-channel = "1.5.1"
+hex = "0.4.3"
```

Comment:
I'm curious - why can't we use external crates in examples? It makes sense to me in the library code, but not in the examples.

Reply:
Know we already spoke on discord, but just to have a record here for completeness: 

We want to stick to zero external dependencies in the protocol libs and applications because of security reasons, with the exceptions of things like tokio or async_std. Another exception is the use of serde in the message-generator. 

A applications in the `examples` directory will likely be moved to the `roles` directory when they become stable enough, so we just want to limit external dependencies from the start.